### DISCUSSION TITLE
What was the first thing you ever developed on GitHub?

### DISCUSSION THREAD

**Discussion by jricciardi** [community/community#74871] [New to GitHub]
What was your very first project?

**Comment by jricciardi** [comment]
Mine: I made a simple SQL database for a T-SQL Foundations course 😄

**Comment by queenofcorgis** [comment]
I made a markdown portfolio as part of a GitHub Learning Lab course (now https://skills.github.com/) :octocat:

**Comment by jennysharps** [comment]
A little class calendar plugin for WordPress written in PHP for my first coding job ever

**Comment by iansan5653** [comment]
🚀 I helped build [the software](https://github.com/usfsoar/NSL_16-17_Navigation) for a competition with my university's rocketry team. The goal was to identify a target on the ground using a camera attached to a payload falling under a parachute. We had a webcam on two servos with software that tried to aim the camera based on orientation & GPS location. It didn't work (the batteries died on the launchpad 😅), but it was so much fun. It wasn't just the first time I used GitHub -- it was also the first time I'd ever written code on a team :octocat: ❤️

**Comment by ghostinhershell** [comment]
I was young and a fan of the show [Serial Experiments Lain](https://www.google.com/search?q=serial+experiments+lain&rlz=1C5GCEM_enUS1079US1079&oq=serial+exp&gs_lcrp=EgZjaHJvbWUqDQgAEAAY4wIYsQMYgAQyDQgAEAAY4wIYsQMYgAQyCggBEC4YsQMYgAQyBggCEEUYOTIHCAMQABiABDIHCAQQLhiABDIHCAUQABiABDIHCAYQABiABDIGCAcQRRhA0gEIMjEzN2owajGoAgCwAgA&sourceid=chrome&ie=UTF-8), so I created [a blog page for the show](https://github.com/ghostinhershell/Lain-Iwakura-Blog) from scratch.

Funny enough, my user handle was inspired by [Ghost in the Shell](https://www.google.com/search?q=ghost+in+the+shell&rlz=1C5GCEM_enUS1079US1079&oq=ghost+in+the+&gs_lcrp=EgZjaHJvbWUqDQgAEAAY4wIYsQMYgAQyDQgAEAAY4wIYsQMYgAQyDQgBEC4Y1AIYsQMYgAQyDQgCEC4Y1AIYsQMYgAQyDQgDEC4Y1AIYsQMYgAQyBggEEEUYPTIGCAUQRRg9MgYIBhBFGD0yBggHEEUYQNIBCDIxNTVqMGoxqAIAsAIA&sourceid=chrome&ie=UTF-8), another show I was a fan of at the time. So I guess my love of anime inadvertently guided me here 🤷🏾‍♀️

**Comment by samus-aran** [comment]
Oh man..... It had to be to create a blog for myself but professionally was contributing to [SuiteCRM](https://github.com/salesagility/SuiteCRM)

**Comment by lclindeman** [comment]
I built a to-do list (of course) in Ruby on Rails (it was 2014! 😁 )

**Comment by gaurav-ketkar** [comment]
Is there a GitHub search query that pulls that up for you? :D 
I want to say mine was a browser extension for an EventBrite interview but I am not sure 🤔

**Reply by jennysharps** [reply]
There probably is, but I just went and looked at all of my repositories and scrolled straight down to the bottom to find the last one with activity on it

**Comment by Bestra** [comment]
My choir had a directory built in MS Access that had suddenly stopped working properly (this was 2011). I was learning Ruby on Rails and needed a portfolio piece to find a junior dev position. I rewrote the directory as a Rails app and deployed it to Heroku 🚀

**Comment by mishmanners** [comment]
I think mine was something to do with the LIFX lights and building a controller for them so I could control them from my Stream Deck while live on Twitch 😋

**Comment by ricardobar96** [comment]
My first assignment for an HTML course, our teacher tasked us to create a character sheet for some random characters from World of Warcraft 😆

**Comment by Akash1134** [comment]
Thank you everyone for contributing to this discussion 🚀 This particular discussion was a part of Universe 2023 that we are now moving over to other relevant categories -- moving this thread over to the `New to GitHub` category. We'd love to hear more from you 🙌

**Comment by marcosfa00** [comment]
I develop a website using **HTML5** & **CSS** it was my christmas project of my technical IT course
